rsyslog-logger.go

```diff
@@ -44,13 +43,24 @@ func (r *Rsyslog) AddTask(taskInfo TaskInfo) error {
 		fmt.Printf("[ERROR] %v\n", err)
 		return err
 	}
-	// TODO - Support multiple file configurations
-	configFileLocation := fmt.Sprintf("%s/%s-%s.conf", r.ConfigLocation, CommonPrefixToConfigFiles, taskInfo.TaskID)
+
+	// We create symlink of the tasks' sandbox dir so that,
+	// we can reduce the length of the file path that we provide as
+	// part of configs in rsyslog's imfile directive.
+	// When the full file path > 200, rsyslog crashes with "buffer overflow" (evil smile)
+	err = os.Symlink(taskInfo.CWD, fmt.Sprintf("%s/%s", r.WorkDir, taskInfo.TaskID))
```

Comment:
@aayushKumarJarvis Can we please extract `fmt.Sprintf("%s/%s", r.WorkDir, taskInfo.TaskID)` into a private function inside `Rsyslog` ? We're using it at another place and it's best if the whole thing is at a single place.